lang/go/bindings/cne/msgchan.go

```diff
@@ -124,3 +135,40 @@ func (mc *MsgChannel) SendFree() int {
 
 	return int(sndFree)
 }
+
+func (mc *MsgChannel) Info() *MsgChannelInfo {
+	var mcInfo C.msgchan_info_t
+
+	if mc == nil {
+		return nil
+	}
+	if C.mc_info(mc.mChan, &mcInfo) == -1 {
+		return nil
+	}
+
+	info := &MsgChannelInfo{
+		RxRing:       mcInfo.recv_ring,
+		TxRing:       mcInfo.send_ring,
+		ChildCount:   int(mcInfo.child_count),
+		SendCalls:    uint64(mcInfo.send_calls),
+		SendCnt:      uint64(mcInfo.send_cnt),
+		RecvCalls:    uint64(mcInfo.recv_calls),
+		RecvCnt:      uint64(mcInfo.recv_cnt),
+		RecvTimeouts: uint64(mcInfo.recv_timeouts),
+	}
+
+	return info
+}
+
+func (mc *MsgChannel) Pointers() (rx unsafe.Pointer, tx unsafe.Pointer) {
```

Comment:
Why do you need this API? Do you want to test it in the cne_test.go?

Reply:
Added the test for this one.